wallet/db.c

```diff
@@ -668,6 +668,7 @@ static struct migration dbmigrations[] = {
     {SQL("ALTER TABLE invoices ADD COLUMN local_offer_id BLOB DEFAULT NULL REFERENCES offers(offer_id);"), NULL},
     /* A reference into our own offers table, if it was made from one */
     {SQL("ALTER TABLE payments ADD COLUMN local_offer_id BLOB DEFAULT NULL REFERENCES offers(offer_id);"), NULL},
+    {SQL("ALTER TABLE channels ADD tx_sigs_remote INTEGER DEFAULT 0;"), NULL},
```

Comment:
Can we make this `funding_tx_remote_sigs_received`, otherwise I know I'll forget this eventually :-)

Reply:
The only exception to this should be if the field is conditionally set with some other field, in which case grouping makes sense.